### PR TITLE
Update changelog.md [3.0.0]

### PR DESCRIPTION
**Pull request by krzemin**

### CODE REVIEW

File: _pages/changelog.md [hunk 1]
```diff
@@ -5,7 +5,17 @@ include_in_header: true
 ---
 # Change Log
 
-## Release 2.4.2 `latest`
+## Release 3.0.0 `latest`
+**Jan 25, 2024**
+
+- Added support for multiple MPD server connections
+- Automatic discovery of local MPD instances using Bonjour/Zeroconf protocol
+- Ability to define multiple local playback stream URLs (with various bitrate and audio quality)
```
Comment (NoemiRozpara) [inline _pages/changelog.md:13]:
```suggestion
- Ability to define multiple local playback stream URLs (with various buffer sizes)
```

File: _pages/changelog.md [hunk 2]
```diff
@@ -5,7 +5,17 @@ include_in_header: true
...
+- Improved compatibility with Mopidy server
+- Local playback: fixed crashes when playing FLAC streams
+- Numerous bug fixes and stability improvements
+
```
Comment (NoemiRozpara) [inline _pages/changelog.md:18]:
```suggestion
- Fixed navigation crash on iOS 17

```

Comment (krzemin) [reply _pages/changelog.md:18]: Isn't it contained within `Numerous bug fixes and stability improvements`?

File: _pages/changelog.md (hunk 1 above)
Comment (krzemin) [reply _pages/changelog.md:13]: Yes, but buffer size is only a tech detail of our configuration model. What is important for users is that they can define different MPD streaming outputs with various bitrate/quality. IMO better for marketing and more communicative for users outside of our detailed config context.

File: _pages/changelog.md [hunk 3]
```diff
@@ -5,7 +5,17 @@ include_in_header: true
...
 
-## Release 2.4.2 `latest`
+## Release 3.0.0 `latest`
+**Jan 25, 2024**
+
+- Added support for multiple MPD server connections
+- Automatic discovery of local MPD instances using Bonjour/Zeroconf protocol
+- Ability to define multiple local playback stream URLs (with various bitrate and audio quality)
+- Improved compatibility with Mopidy server
+- Local playback: fixed crashes when playing FLAC streams
```
Comment (NoemiRozpara) [inline _pages/changelog.md:15]:
```suggestion
- Local playback: fixed crashes when playing FLAC streams
- Fixed navigation crash on iOS 17
```

**Pull request merged**